src/tophu/_io.py

```diff
@@ -564,6 +566,7 @@ def __init__(
             crs=crs,
             transform=transform,
             dtype=dtype,
+            options=options,
```

Comment:
Shouldn't we be unpacking the `options` dict into `rasterio.open()` instead, like this?

```suggestion
            **options,
```

The [docs](https://github.com/rasterio/rasterio/blob/5e001a08beb747fef0b60b5ccdbf2e5a08da7e05/docs/topics/image_options.rst#creation-options) that you linked in the PR description and the API docs [here](https://rasterio.readthedocs.io/en/stable/api/rasterio.html#rasterio.open) both seem to suggest that each individual option should be passed to `rasterio.open()` as a separate keyword.

Reply:
shoot thanks, I think I did that the first time then messed it all up in my branch rebase